credmark/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = '0.8.22'
```

Comment:
This __init__,py cannot be in this namespace folder. It must be in credmark/cmf

Reply:
Let's do the new version handling in a separate PR and keep this PR for only Jupyter support.

Reply:
Let's do the new version handling in a separate PR and keep this PR for only Jupyter support.

Reply:
yes. removed __init__.py from this branch.